e2e_testing/torchscript/nll_loss.py

```diff
@@ -50,8 +49,7 @@ def __init__(self):
   ])
   # None of the index is ignored here, since the ignored index is out of bounds.
   def forward(self, x, y):
-    return torch.ops.aten.nll_loss_forward(self=x,
-                                           target=y,
+    return torch.ops.aten.nll_loss_forward(target=y,
```

Comment:
oh, I should've made this clearer. I meant replacing `self=x` with `x`

Reply:
ok